config/defaults.go

```diff
@@ -80,6 +86,10 @@ func (d *Defaults) Validate() error {
 		}
 	}
 
+	if d.KillAfter < 0 {
+		return errors.Errorf("killAfter must not be negative")
```

Comment:
This provides validation at runtime that the KillAfter field is not negative. Should there also be validation before deploying and starting the controller (at configure time)?

Reply:
I think we called defs.Validate() to run the check at the configure time. I think I did not understand your question fully.